pkg/notes/maps/testdata/fullmap.yaml

```diff
@@ -0,0 +1,28 @@
+---
+pr:  123
+commit: 1a89038915fe77d73bf7c9cfa8f2ce123a464c82
+release-note:  
+  text: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+  author: kubernetes-ci-robot
+  areas:
+    - testarea
+  kinds:
+    - documentation
+  sigs:
+    - api-machinery
+  feature: true
+  action_required: false
+  release_version: v1.18.4
+  documentation:
+    - description: Test documentation
+      url: https://wkdkjdjkdkfj
+      type: kep
+datafields:
```

Comment:
What if we omit that additional field and move `cve` to the root level?

Reply:
I added that one to be able to have everything bellow `datafields: ` be read as an empty interface. That way we do not have to enforce a structure to those data fields. The idea was to enable users to add any free-form info to the release note (especially since more people are using the tools now). Do you think this is a good feature to have?

Then, I think that If I move the `cve` field to the root we would have to predefine that struct (and all others). Or maybe there is a way to do it that I have not yet considered?

Reply:
Okay cool makes sense, sounds good to me to leave it as `datafields`. :+1: